### ISSUE TITLE
Add documentation for creating/maintaining packages

### ISSUE THREAD

**Issue by agaffney** [blinklabs-io/cardano-up#168]
Describe the package format, how flags, dependencies, etc. work

**Comment by fabianbormann** [comment]
Same here. I would like to add [yaci-devkit](https://github.com/bloxbean/yaci-devkit/blob/main/docker-compose.yml) to cardano-up. 

It is a local cardano testnet including dev tools like a lightweight developer-friendly explorer. 

Maybe it is because I used go last time in 2018 but I fail currently in adding the package. In my fork I added the yaml file and built the project locally but the local build of `./cardano-up list-available` doesn't list yaci-devkit `./cardano-up validate ../../packages/yaci-devkit` returns `No problems found!`.

A section in the README like requested from @agaffney would be very helpful.
